packages/web/config/ibc-assets.ts

```diff
@@ -1813,6 +1813,13 @@ export const IBCAssetInfos: (IBCAsset & {
         destChannelId: "channel-0",
         coinMinimalDenom: "ulore",
       },
+      {
+        counterpartyChainId: "pirin-1",
+        sourceChannelId: "channel-783",
+        destChannelId: "channel-0",
+        coinMinimalDenom: "unls",
+        isVerified: true,
```

Comment:
I don't recall this ever being approved. Please remove isVerified

Reply:
removed